ocs_ci/framework/conf/default_config.yaml

```diff
@@ -235,7 +235,7 @@ EXTERNAL_MODE:
 UI_SELENIUM:
   browser_type: "chrome"
   chrome_type: "google-chrome"
-  headless: True
+  headless: False 
```

Comment:
Why did you change the headless to False?
We must to run it with headless on jenkins slave..

Reply:
@OdedViner sorry, I did it for testing but forgot to revert it